app/setup_handlers.go

```diff
@@ -1,32 +1,12 @@
 package app
 
 import (
-	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
 )
 
-const upgradeName = "0.9.8"
```

Comment:
Only change version number when doing a an upgrade in a separate PR for the deployment